src/frontend/packages/core/src/core/permissions/current-user-permissions.config.ts

```diff
@@ -6,7 +8,6 @@ export interface IPermissionConfigs {
 export type PermissionTypes = string;
 export type CurrentUserPermissions = string;
 export type ScopeStrings = string;
-export type PermissionValues = string;
```

Comment:
Just a comment, we may find these other types will be referenced by plugins such as `cloud-foundry`. It might make sense at that point to group them all together in an types file that exposes all